catalogs/requirements-conda.txt

```diff
@@ -8,3 +8,4 @@ rasterio==1.1.5
 requests==2.24.0
 shapely==1.7.0
 scikit-learn==0.23.2
+python-dateutil==2.8.1
```

Comment:
Existing deps look to be in alphabetical order, re-sort to maintain consistency.